docs/reference/commandline/dockerd.md

```diff
@@ -68,7 +68,8 @@ Options:
       --max-concurrent-downloads int          Set the max concurrent downloads for each pull (default 3)
       --max-concurrent-uploads int            Set the max concurrent uploads for each push (default 5)
       --metrics-addr string                   Set address and port to serve the metrics api (default "")
-      --mtu int                               Set the containers network MTU
+      --mtu int                               Set the containers network mtu
+      --no-new-privileges                     Disable container processes from gaining new privileges
```

Comment:
sorry I just mean in this section https://github.com/docker/docker/blob/master/docs/reference/commandline/dockerd.md#linux-configuration-file it needs to be added in the example config file.